metrics/web/lib/dashboard/presentation/widgets/projects_search_input.dart

```diff
@@ -7,22 +7,44 @@ import 'package:metrics/common/presentation/strings/common_strings.dart';
 import 'package:metrics/common/presentation/widgets/metrics_text_form_field.dart';
 
 /// [TextField] with the ability to search across projects.
-class ProjectSearchInput extends StatelessWidget {
+class ProjectSearchInput extends StatefulWidget {
```

Comment:
📱 
This widget looks more like a common widget. Maybe we can move it to the `common` package? Please, contact me for more details on this.